crates/test_utils/wasm/build.rs

```diff
@@ -81,7 +81,10 @@ fn build_test_wasms(
         if should_build {
             cmd.stdout(Stdio::piped());
             cmd.stderr(Stdio::piped());
-            cmd.env("RUSTFLAGS", "-C opt-level=z");
+            cmd.env(
+                "RUSTFLAGS",
+                "-C opt-level=z --cfg getrandom_backend=\"custom\"",
```

Comment:
This is the bit that WASM projects will have to do when upgrading to 0.6. It can be put in the manifest but I thought this was tidier to have the build tool do it for all our WASMs